packages/fiori-freestyle-writer/src/defaults.ts

```diff
@@ -45,6 +45,17 @@ export function setDefaults(ffApp: FreestyleApp<unknown>): void {
     }
     // All fiori-freestyle apps should use load reuse libs, unless explicitly overridden
     ffApp.appOptions = Object.assign({ loadReuseLibs: true }, ffApp.appOptions);
+    if (ffApp.ui5) {
```

Comment:
Why is this necessary here but not in fiori-elements-writer?

Reply:
But there is something similar in FE https://github.com/SAP/open-ux-tools/pull/2280/files#diff-e26bb8b0304f6b67d4b5d0479d6c75263405883302396f777dd7f1ac6cb37211R35

Reply:
Same outcome but I think a clearer implementation. We can take that as a separate alignment task.